mini-project/BillManagementSystem/accounting/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
 
 from . import views
+from strawberry.django.views import AsyncGraphQLView
+from .schema import schema
 
 urlpatterns = [
-    path('', views.index, name='index')
+    path('', views.index, name='index'),
+    path('graphql', AsyncGraphQLView.as_view(schema=schema)),
```

Comment:
For your current situation, don't use async.
It requires a deeper understanding of django, resolvers and the event loop. Debug messages would also be much different and difficult to read.